loadtest/control/gencontroller/actions.go

```diff
@@ -33,6 +33,14 @@ func logout(u user.User) control.UserActionResponse {
 	return control.UserActionResponse{Info: "logged out"}
 }
 
+func (c *GenController) login(u user.User) control.UserActionResponse {
+	res := control.Login(u)
+	if res.Err == nil && res.Warn == "" {
+		st.incUsers()
+	}
```

Comment:
What would happen if somehow one of the users fails to login? Is the action being repeated or we end up stuck waiting forever?

Reply:
The latter. We don't have target checks for user logins, that's why I kept it as a single counter. But I realize that one failure, and we end up with an infinite loop.

Do you mind adding a fix? It would be just about converting this field to a StateTarget and changing the actions to a loop. Or something close to that.

Reply:
Okay, pushed the changes.